terraform/modules/cluster/addons.tf

```diff
@@ -431,14 +413,10 @@ module "eks_blueprints_kubernetes_addons" {
 module "eks_blueprints_ack_addons" {
   source = "github.com/aws-ia/terraform-aws-eks-ack-addons?ref=v1.1.0"
 
-  depends_on = [
```

Comment:
the module waits for the nodegroups, which they wait for the VPC CNI due to the `before_compute` which means this explicit dependency can be removed